ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,322 @@
+    margin-right: ${spacing.sp8};
+    color: ${(props) => props.theme.textSecondary};
+    text-align: right;
+  }
+`;
+
+const NonHealthyPopUpTitle = styled.div`
+  font-weight: ${fontWeight.bold}
+  text-align: center;
+`;
+
+const NonHealthyPopUpItem = styled.div`
+  width: 100%;
+  display: flex;
+  margin: ${spacing.sp4} ${spacing.sp14};
+`;
+
+const NonHealthyPopUpSeverity = styled.div`
+  color: ${(props) => {
+    const theme = props.theme;
+    let color;
+
+    switch (props.status) {
+      case STATUS_HEALTH:
+      case STATUS_SUCCESS:
+        color = theme.statusHealthy;
+        break;
+      case STATUS_WARNING:
+        color = theme.statusWarning;
+        break;
+      case STATUS_CRITICAL:
+        color = theme.statusCritical;
+        break;
+      default:
+        color = theme.statusHealthy;
+    }
+    return color;
+  }};
+`;
+
+const ClickableIcon = styled.i`
+  self-align: flex-end;
+`;
+
+const ServiceItem = ({
+  label,
+  status,
+  alerts,
+}: {
+  label: string,
+  status: string,
+  alerts: Alert[],
+}) => {
+  let date = null;
+
+  const getStartDate = (alerts: Alert[]) => {
+    // Check we only have alerts and set startsAt properties
+    const filtered = alerts.filter((item) => item && item.startsAt);
+    // Sort to make sure we have oldest alert first
+    filtered.sort((a, b) => new Date(a.startsAt) - new Date(b.startsAt));
+
+    if (filtered[0] && filtered[0].startsAt) return filtered[0].startsAt;
+    return '';
+  };
+
+  if (alerts.length) date = new Date(getStartDate(alerts));
+
+  if (!alerts.length && status === STATUS_HEALTH)
+    return (
+      <ServiceItemElement aria-label={label}>
+        <ServiceItemLabelWrapper>
+          <CircleStatus status={status} />
+          <ServiceItemLabel>{label}</ServiceItemLabel>
+        </ServiceItemLabelWrapper>
+      </ServiceItemElement>
+    );
+  else
+    return (
+      <NonHealthyServiceItemElement>
+        <Tooltip
+          // Right placement to avoid Z index issues between left sidebar and tooltip or out of screen tooltip displays
+          placement="top"
+          overlayStyle={{
+            // em sizing to handle font-size change on large screens
+            width: '20em',
+            height: '100%',
+          }}
+          overlay={
+            <NonHealthyPopUp>
+              <NonHealthyPopUpTitle>View details</NonHealthyPopUpTitle>
+              <NonHealthyPopUpItem>
+                <label>Severity</label>
+                <NonHealthyPopUpSeverity status={status}>
+                  {status}
+                </NonHealthyPopUpSeverity>
+              </NonHealthyPopUpItem>
+              {
+                // Checking date validity
+                date instanceof Date && !isNaN(date) && (
+                  <NonHealthyPopUpItem>
+                    <label>Start</label>
+                    <div>
+                      {`${date.toISOString().split('T')[0]}
+              ${date.toISOString().split('T')[1].slice(0, 8)}`}
```

Comment:
I think second is not necessary.
so you could benefit from`formatDateToMid1` in utils.js